code/modules/overmap/ships/computers/sensors.dm

```diff
@@ -640,3 +640,16 @@
 		/obj/item/bluespace_crystal,
 		/obj/item/stack/cable_coil = 30
 	)
+
+/obj/machinery/shipsensors/strong/venator/relay
```

Comment:
should be `/obj/machinery/shipsensors/strong/relay`
venator sensors are specific to horizon
and this subtype overrides all the vars anyways

Reply:
should also have `component_types` and its own circuit board

Reply:
it should also not scan further out than Horizon's sensors
horizon's sensors are supposed to be the best of the best, most high tech, etc
meanwhile some tiny old sensors on a tiny old station are better?

Reply:
I agree with this, the Horizon is supposed to have the most power sensors around. It was one of the selling points for the initial lore arc when it launched.